<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Type</class>
 <widget class="QWidget" name="Type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Type</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Cpu_hinh.qrc">
    <normaloff>:/hinh/image/link.png</normaloff>:/hinh/image/link.png</iconset>
  </property>
  <widget class="QLabel" name="logoLabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius: 999px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Cpu_hinh.qrc">:/hinh/image/logo1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_back_to_Manager">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="Cpu_hinh.qrc">
     <normaloff>:/hinh/image/exit (1).png</normaloff>:/hinh/image/exit (1).png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>571</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox {
    border: 2px solid #0078D7; /* Độ dày và màu viền */
    border-radius: 5px;        /* Bo góc nhẹ nếu muốn */
    margin-top: 2px;
         /* Khoảng cách giữa tiêu đề và nội dung */
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="title">
    <string>ADD Processor</string>
   </property>
   <widget class="QLineEdit" name="le_processorname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Processor Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_processor_id">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Auto</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_addprocess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="icon">
     <iconset resource="Cpu_hinh.qrc">
      <normaloff>:/hinh/image/add.png</normaloff>:/hinh/image/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel_processor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>113</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="Cpu_hinh.qrc">
      <normaloff>:/hinh/image/paper.png</normaloff>:/hinh/image/paper.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Processor ID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_deleteprocess">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>108</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset resource="Cpu_hinh.qrc">
      <normaloff>:/hinh/image/trash.png</normaloff>:/hinh/image/trash.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_change_processor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>117</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
    <property name="icon">
     <iconset resource="Cpu_hinh.qrc">
      <normaloff>:/hinh/image/editing.png</normaloff>:/hinh/image/editing.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Processor Name:</string>
    </property>
   </widget>
   <zorder>le_processorname</zorder>
   <zorder>le_processor_id</zorder>
   <zorder>pb_addprocess</zorder>
   <zorder>pb_cancel_processor</zorder>
   <zorder>pb_deleteprocess</zorder>
   <zorder>pb_change_processor</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>891</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color: #1E1E2F;
background-color: qradialgradient(spread:pad, cx:0.508, cy:0.505818, radius:0.295, fx:0.505, fy:0.517045, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(7, 6, 10, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>571</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox {
    border: 2px solid #0078D7; /* Độ dày và màu viền */
    border-radius: 5px;        /* Bo góc nhẹ nếu muốn */
    margin-top: 2px;
         /* Khoảng cách giữa tiêu đề và nội dung */
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="title">
    <string>ADD Processor</string>
   </property>
   <widget class="QTableWidget" name="table_processor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Processor ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Processor Name</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_exit_pro">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(0, 255, 255);
     color:black;
    border: 1px solid black;     /* viền đen */
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffffcc,  /* vàng trắng nhạt */
        stop:1 #fff176   /* vàng nhạt */
    );
    color: black;
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #ffb74d,  /* cam nhạt */
        stop:1 #f57c00   /* cam đậm */
    );
    color: black;
}
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="Cpu_hinh.qrc">
     <normaloff>:/hinh/image/reject.png</normaloff>:/hinh/image/reject.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Cpu_hinh.qrc">:/hinh/image/link.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);
border-radius:10px;
    border: 1px solid black; </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>logoLabel_2</zorder>
  <zorder>groupBox_7</zorder>
  <zorder>pb_back_to_Manager</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>pb_exit_pro</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="Cpu_hinh.qrc"/>
 </resources>
 <connections/>
</ui>
